src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/RunnableProjectGenerator.cs

```diff
@@ -413,12 +422,11 @@ internal static object InternalConvertParameterValueToType(IEngineEnvironmentSet
         // checks that all the template sources are under the template root, and they exist.
         internal bool AreAllTemplatePathsValid(IRunnableProjectConfig templateConfig, RunnableProjectTemplate runnableTemplate)
         {
-            ITemplateEngineHost host = runnableTemplate.Source.EnvironmentSettings.Host;
+            ILogger logger = runnableTemplate.Source.EnvironmentSettings.Host.Logger;
```

Comment:
@bozturkMSFT  it appears we already have some template validation in `AreAllTemplatePathsValid` and `PerformTemplateValidation` however output is done to `debug` that's why i never saw it before.

Reply:
Imo lets promote this to Warning/Error levels.

Reply:
I agree, however I would extract it to separate issue for .NET 7.
reason: SDK templates produce a lot of warnings (see above), we need to fix them prior to promoting.

Reply:
What about Errors?(lets keep Warnings to Debug for now)
Errors afaiu, means template is not loaded and author has no idea why...

Reply:
https://github.com/dotnet/templating/issues/3147